index.js

```diff
@@ -9,7 +6,14 @@ module.exports = robot => {
     const issueOwner = issue.user.login;
     const repo = context.repo({username: issueOwner});
 
-    const config = await configs(context, './lib/defaults');
+    let config;
+
+    try {
+      config = await context.config('teacherbot.yml');
+    }
+      catch(err) {
+        config = Object.assign( {}, require('./lib/defaults.js') || {} );
```

Comment:
Thinking of also reducing complexity and including the defaults inline:

```js
    try {
      config = await context.config('teacherbot.yml');
    }
      catch(err) {
        config = Object.assign( {},
          {
            addCollaborators: {
              newCollaboratorMessage: 'Hi! I\'m the friendly :robot: of this repo.\n\nWe\'re happy you\'re here :wave:. Everyone is welcome here, so I\'m making you a collaborator. This will give you access to commit on this repo.',
              existingCollaboratorMessage: 'Hi! I\'m the friendly :robot: of this repo.\n\nI can see that you\'re already a collaborator. Any other issues are above my paygrade at the moment, so we\'ll have to wait for a pesky hu-man. Not to worry though, they\'ll drop by within 24 hours to answer your questions!'
            }
          } || {} );
      }
```